MFVNeutralino/python/EventHistos_cfi.py

```diff
@@ -2,5 +2,6 @@
 
 mfvEventHistos = cms.EDAnalyzer('MFVEventHistos',
                                 mevent_src = cms.InputTag('mfvEvent'),
+                                vertex_src = cms.InputTag('mfvSelectedVerticesTight'),
```

Comment:
Correct me if I'm wrong: do we need to make sure that histos.py sets the correct vertex_src for each trk multiplicity here? Otherwise I think even when we look e.g. at the folders for Ntk = 3, the corresponding vertex-level histograms in EventHistos will also be of the 5-trk vertices?

Reply:
This part is irrelevant to the folder with the mfvSelectedVerticesTight cuts applies. I add vertex-related objects in EventHistos.cc in order to make h_nsv  plots at the event-level histograms. Here https://github.com/DisplacedVertices/cmssw-usercode/blob/d2c0260e403737b9440bb2850242f40d7f7dd3b4/MFVNeutralino/plugins/EventHistos.cc#L360-L368

Reply:
I think that's my point though--that if we are looking at the EventHistos for (e.g.) the set of events with one 4-trk vertex, the plot will still be of 5-trk vertices. 

To be concrete: if you look at
`/uscms/home/pkotamni/nobackup/crabdirs/HistosNoBTagULrequestV27Bm/ggHToSSTodddd_tau10mm_M55_2017.root`

and compare the `h_nsv` plot from `Ntk4mfvEventHistosOnlyOneVtx` with that in `Ntk4mfvVertexHistosOnlyOneVtx`, you can see that the "number of vertices" differs. But since the same selection is applied to make each of these folders, the difference must be due to the definition of "vertex" used as input to making the plot. I think it will be very confusing if these two plots, with the same name, show different results.

Similarly, between the two example folders above, `h_svntracks` (from the EventHistos) and `h_sv_all_ntracks` (from the VertexHistos) differ, which seems unexpected given the similar names and the identical selection applied...

Reply:
Done (?) I revert EventHistos.cc + its cfi back to its master version and rename PreSel to EvtFilt(VtxSel) to make it less confusing.

Reply:
Thanks! EventHistos.cc looks good. 

For PreSel -> EvtFilt, I remember now that [the presel](https://github.com/DisplacedVertices/cmssw-usercode/blob/clean_vertexing/MFVNeutralino/python/AnalysisCuts_cfi.py#L6) _is_ applied as well, right? i.e. the offline HT requirements, and others for bjet triggers.

Maybe we call it PreSelEvtFilt, to make it clear that it is both? (and keep VtxSel for those where relevant)

Reply:
Yes-- you're right! we apply presel/analysis cuts in used-to-be PreSel folders. Sounds good. I can add PreSel to both.

Reply:
Done.